### ISSUE TITLE
Windows: Question for adminrights to execute bat Script

### ISSUE THREAD

**Issue by oliuo1** [rustdesk/rustdesk#2931]
**Describe the bug you encountered:**
see screenshot:
![image](https://user-images.githubusercontent.com/122091080/214295221-facbe1d2-5955-4c0a-ba5a-7ea7c122ef4a.png)

Scenario: 
Own Rustdesk Server with Encryption
Computer A: Win11 Home
Computer B: Win11 Home
On A and B latest nightly (or 1.1.9). 
If i make a Connection from A to B everything is nice working
If i make a connection from B to A immediatly the window seen in the screenshot is appearing. i didnt press "JA", if i press "NEIN" (no) then the windows desktop appears and B has to put in the password for connecting. No button on A for letting B in...

**What did you expect to happen instead?**
connect to A without that window in the screenshot. not everyone a user connects to has admin rights/admin passwort to press "Yes".

**How did you install `RustDesk`?**
2 portables

---

**RustDesk version and environment**
own rustdesk server

**Comment by rustdesk** [comment]
where do you download rustdesk? we do not have the file in your picture.

**Comment by rustdesk** [comment]
I found the file, let me check more.

**Comment by rustdesk** [comment]
Are you using privacy screen?

**Comment by rustdesk** [comment]
@fufesou please check this? this behavior seems unfriendly no matter if privacy screen is used or not.

**Comment by rustdesk** [comment]
> privacy screen
@fufesou obviously the user is not using privacy, then no reason to invoke broker etc.

**Comment by fufesou** [comment]
@oliuo1  Thanks for your feedback.

https://github.com/rustdesk/rustdesk/pull/2954
This prompt window is unneccesary. I've tried to fix this issue.

Would you please comfirm tomorrow? Thx

**Comment by fufesou** [comment]
@oliuo1 Hi, have you ever input and remember the password? You can see "Unremenber" option if the password is remembered.
![1675298782000](https://user-images.githubusercontent.com/13586388/216203099-be47271c-f9bb-485c-8015-b8bbcba200d5.png)

> From A (has admin rights) to B (User has no admin rights) the small window appears on B where i can adjust what A can do. But there is no button to let A in. If A does not know the passwortd the user sitting in front of B cannot let A in.

I've tested this case. I can see the prompt window and "Accept" button.
https://user-images.githubusercontent.com/13586388/216203339-aead666b-58c8-45e9-b715-02136997cf1b.mp4

**Comment by test2a** [comment]
@fufesou this should be an option for the recipient as well. 

If I somehow get connected to a machine, the other side does not know that I have clicked on "remember password". Now I can log in or out without informing the user. 

Yes the rustdesk widget shows up for a second but still.

**Comment by yuliu** [comment]
The bat file re-appears when both ends using 1.2.3-1. The file content of `RustDesk_update_broker.bat` is
```bat


        chcp 65001
        taskkill /F /IM RuntimeBroker_rustdesk.exe
        copy /Y "C:\Windows\System32\RuntimeBroker.exe" "C:\Program Files\RustDesk\RuntimeBroker_rustdesk.exe"
    
if exist "C:\Users\xxx\AppData\Local\Temp\RustDesk_update_broker.bat.undone" del /f /q "C:\Users\xxx\AppData\Local\Temp\RustDesk_update_broker.bat.undone"
```

**Comment by fufesou** [comment]
I've removed it in the nightly build.

https://github.com/rustdesk/rustdesk/discussions/6345

https://github.com/rustdesk/rustdesk/pull/6361

**Comment by yuliu** [comment]
Oh I see. Vers 1.2.3-x just fixes a vuln in 1.2.3 without having any new commits merged since that ver.
